dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/AbstractTrackerPersister.java

```diff
@@ -127,23 +128,24 @@ public TrackerTypeReport persist(EntityManager entityManager, TrackerBundle bund
         //
         // Save or update the entity
         //
-        if (isNew(bundle.getPreheat(), trackerDto)) {
+        if (bundle.getStrategy(trackerDto) == TrackerImportStrategy.CREATE) {
           entityManager.persist(convertedDto);
           typeReport.getStats().incCreated();
           typeReport.addEntity(objectReport);
           updateAttributes(
               entityManager, bundle.getPreheat(), trackerDto, convertedDto, bundle.getUser());
         } else {
-          if (isUpdatable()) {
+          if (trackerDto.getTrackerType() == TrackerType.RELATIONSHIP) {
```

Comment:
how does this influence the bug we've been seeing with someone trying to import events and only getting an ignored count with no validation errors in the report?

Reply:
Here is a case where we are increasing the ignored stats. In my mind if we are fixing the bug making the stats an invariant that will force us to declare an error/warning at the moment of increasing the ignored stat, it will improve the code here as well.